Web Survey on 20211231

MingcongBai opened the pull request:
Topic Description
-----------------

Survey for updates in the `base-web` and `extra-web` categories.

Package(s) Affected
-------------------

- `curl+32` v7.80.0
- `hugo` v0.91.2
- `nginx` v1.21.5
- `qownnotes` v21.12.8
- `seamonkey` v2.53.10.2
- `telegram-desktop` v3.4.0

Security Update?
----------------

Yes

Build Order
-----------

```
extra-optenv32/curl+32
extra-web/hugo
extra-web/nginx
extra-web/qownnotes
extra-web/seamonkey
extra-web/telegram-desktop
```

Test Build(s) Done
------------------

**Primary Architectures**

- [x] AMD64 `amd64`   
- [x] AArch64 `arm64`
- [x] 32-bit Optional Environment `optenv32`

**Secondary Architectures**

Architectural progress for "secondary," or experimental ports does not impede on merging of this topic.

- [x] Loongson 3 `loongson3`
- [x] PowerPC 64-bit (Little Endian) `ppc64el`

Update(s) Uploaded to Stable
----------------------------

**Primary Architectures**

- [ ] AMD64 `amd64`   
- [ ] AArch64 `arm64`
- [ ] 32-bit Optional Environment `optenv32`

**Secondary Architectures**

Architectural progress for "secondary," or experimental ports does not impede on merging of this topic.

- [ ] Loongson 3 `loongson3`
- [ ] PowerPC 64-bit (Little Endian) `ppc64el`

MingcongBai committed:
nginx: update to 1.21.5

MingcongBai committed:
qownnotes: update to 21.12.8

MingcongBai committed:
hugo: update to 0.91.2

MingcongBai committed:
seamonkey: update to 2.53.10.2

MingcongBai committed:
curl+32: update to 7.80.0

MingcongBai committed:
telegram-desktop: update to 3.4.0

WhiredPlanck commented on extra-web/telegram-desktop/spec:
```diff
@@ -1,9 +1,9 @@
-CHKSUMS="sha256::72c51d0f24c78f9362c334091fb2de71ecfe786e8d9076f9f4fca4925809489b \
+CHKSUMS="sha256::8bd9067ba0a044789aa28b321715ba60971dc27ae8d027ef1f9879093c331157 \
          SKIP"
 SUBDIR=tdesktop-$VER-full
```
Shall we quote the path here ...?

MingcongBai replied:
Yes.

MingcongBai replied:
Addressed. Resolving.

MingcongBai committed:
telegram-desktop: quote SUBDIR

WhiredPlanck commented:
Telegram LGTM on AMD64

MingcongBai commented:
![image](https://user-images.githubusercontent.com/5006263/147813984-cb9712f2-6988-4e9d-b302-9aa0f1c10e8c.png)

QOwnNotes requires intervention, we should opt-out by default.

MingcongBai commented:
Other packages LGTM.

WhiredPlanck committed:
qownnotes: update to 22.1.0

WhiredPlanck committed:
qownnotes: use ABTYPE instead of custom build script

- Remove overridden desktop file because build system will install it.
- Write release vendor information to release.h in prepare.
- Move language files to correct path in beyond.

WhiredPlanck committed:
qownnotes: add patch to disable app metrics and heartbeat by default

WhiredPlanck committed:
qownnotes: replace mv with install command in beyond (polish)

WhiredPlanck commented:
QOwnNotes LGTM on AMD64
App metrics and heartbeat are disabled by default.
All architecture test builds done.

![图片](https://user-images.githubusercontent.com/47623588/147854815-c7770bf0-4b13-4ef9-b5b5-ffa9d3a9d307.png)
![图片](https://user-images.githubusercontent.com/47623588/147854882-8fcbe741-f15e-4d26-b2fc-35a4bec4118a.png)
![图片](https://user-images.githubusercontent.com/47623588/147854960-224defbf-d198-47ae-b045-b0af07a110ae.png)
